.env.example

```diff
@@ -1,6 +1,4 @@
 DATABASE_URL=postgres://postgres:{PASSWORD}@{HOST}:5432/postgres
```

Comment:
would you care to change this URL to something that you would use to connect to the docker instance? no need to keep the curly braces ig.